spot_rl_experiments/spot_rl/utils/img_publishers.py

```diff
@@ -666,10 +666,12 @@ def inference(self, hand_rgb, timestamp, stopwatch):
         # We decide to hardcode these classes first as this will be more robust
         # and gives us the way to control the detection
         # multi_classes = [["ball", "cup", "table", "cabinet", "chair", "sofa"]]
-        multi_classes = [["glass bottle"]]
-        self.owlvit.update_label(multi_classes)
+        multi_classes = rospy.get_param("/multi_class_object_target").split(",")
+        self.owlvit.update_label([multi_classes])
```

Comment:
Here `multi_classes` is already a list right (as we called .split() ) ?
Then we can just pass it to `owlvit.update_label` ?

Reply:
owlvit model needs to be [["cup"], ["apple"]] format, which is a bit weird.